app/views/thesis/process_theses.html.erb

```diff
@@ -199,7 +199,8 @@
                                aria: { describedby: 'thesis_date_ids-hint' }
                                } %>
           <%= f.input :graduation_year, label: 'Year *',
-                                        collection: [Date.today.last_year.strftime('%Y'),
+                                        collection: [Date.today.last_year.last_year.strftime('%Y'),
```

Comment:
Oh good catch, this is a side effect I should have noted explicitly as unrelated. I also think this is a bandaid for a bigger looming problem. I'll open a ticket about solving this more permanently as at some point there will be older theses in the system that we can't process because of this lol.